### DISCUSSION TITLE
在未来是否应该将json文件改成一个包含json的文件夹进而支持多个文件的放入

### DISCUSSION THREAD

**Discussion by Littlefean** [graphif/project-graph#132] [功能请求]
来自群友的想法

```
其實，我在想說 要不要幹脆就把每一份檔案的建構都用一個資料夾
然後呃之後未來，可能就可以直接新增一個檔案 比方說新增一個 md 檔案
然後照片什麼的也可以存在這裏面
```

**Comment by zty012** [comment]
文档就是一个zip文件，后缀名为`pg`，里面有名为`project[n]`的文件，其中n是数字，project文件存储json信息，每个project文件最大1MB（防止文件大写入慢）

**Comment by Littlefean** [comment]
可能如何分割每个json文件，还要确保他们小于1MB是一个麻烦的事情

**Comment by Littlefean** [comment]
archiver这个js库可以操作zip（——2024年11月17日在开发群中的讨论）

**Reply by zty012** [reply]
这个库只能在nodejs用，浏览器环境可以用zip.js
